templates/csharp/api-plugin-from-scratch-bearer/appPackage/apiSpecificationFile/repair.yml

```diff
@@ -0,0 +1,57 @@
+openapi: 3.0.0
+info:
+  title: Repair Service
+  description: A simple service to manage repairs
+  version: 1.0.0
+servers:
+  - url: ${{OPENAPI_SERVER_URL}}/api
+    description: The repair api server
+components:
+  securitySchemes:
+    apiKey:
+      type: http
+      scheme: bearer
+paths:
+  /repair:
+    get:
+      operationId: repair
+      summary: Search for repairs
+      description: Search for repairs info with its details and image
+      parameters:
+        - name: assignedTo
+          in: query
+          description: Filter repairs by who they're assigned to
+          schema:
+            type: string
+          required: false
+      security:
+        - apiKey: []
+      responses:
+        '200':
+          description: A list of repairs
+          content:
+            application/json:
```

Comment:
This format is not correct, please refer to the template on VSC for adjustments. The previous templates on VS had some issues, we'll need to correct them later.

Reply:
Sure.